hack/generator/pkg/astmodel/file_definition.go

```diff
@@ -7,6 +7,7 @@ package astmodel
 
 import (
 	"bytes"
+	"github.com/Azure/k8s-infra/hack/generator/pkg/consts"
```

Comment:
Do we like this pattern?

Reply:
Looks Ok to me. Needs a `// TODO` comment reminding us to update the path when we move the generator.

Reply:
Added the todo.

We don't need another package - we could put this in `astmodel`. 
It's used in both  the `codegen` and `astmodel` packages currently because `codegen` performs the directory scan for deletion, and `astmodel` performs the generation of the files.

`codegen` has a reference to `astmodel` (which is why we could put this there).

I'll go ahead and move this for now (and remove this extra package) - I think it was created because I didn't like `astmodel` being the thing that defines file naming (because then `codegen needs to reach into astmodel to figure that out in order to do deletion), but we can solve that in another change.